typescript/sdk/src/utils/env.ts

```diff
@@ -0,0 +1,13 @@
+import z from 'zod';
+
+import { safelyAccessEnvVar } from '@hyperlane-xyz/utils';
+
+const envScheme = z.object({
+  HYP_KEY: z.string().optional(),
+  ANVIL_IP_ADDR: z.string().optional(),
+  ANVIL_PORT: z.number().optional(),
+});
+
+const parsedEnv = envScheme.safeParse(safelyAccessEnvVar('env', true));
```

Comment:
Hmm, sorry if I wasn't clear before but I don't think this is quite right either. Wouldn't this try to access an env var called `env`? so `process.env['env']`. Was that the intention or are each of the env vars above meant to be set individually?

Reply:
Continuing– it now seems appropriate to encapsulate fork.ts in a class, given that it lives in SDK side + enables IO from CLI, which may need to persist certain values (recently discussed [here](https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/3586#discussion_r1564069199). By doing so, we can expose IP/port setters, which can eventually enable multi-chain support (when multiple Anvil instances will need to run)

cc @yorhodes

Reply:
Just to keep this PR moving, I've captured work to do this here: https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/3652

Will hold off on adding this to the current change